tests/unittests/CoreGraphics.drawing/CTDrawingTests.cpp

```diff
@@ -499,10 +499,18 @@ TEXT_DRAW_TEST_P(Fonts, TestFonts) {
     __DrawLoremIpsum(context, path.get(), keys, values);
 }
 
+// Segoe UI is noticeably different on ARM, but we still want to test other fonts so only remove Segoe
+#ifdef WINOBJC_DISABLE_TESTS
```

Comment:
what is the long term plan for this?
are we going to figure out a way to still validate it on ARM (fix the issue?)?
If so could you open a task and put the number here.